english/dragon-slave/dragon-slave.md

```diff
@@ -0,0 +1,10 @@
+A wide-eyed beast looms over head. It is covered with thick plated scales that threaten death with every glint of the sun. It slowly inhales, its broad chest expanding across the sky, blotting your hope of survival.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.